standard-cluster-services/src/main/java/org/terracotta/server/ServerMBean.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Entity API.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.server;
+
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+
+/**
+ *
+ */
+public interface ServerMBean {
+
+  public static ObjectName createMBeanName(String name) throws MalformedObjectNameException {
+    return createMBeanName("org.terracotta", name);
+  }
+
+  public static ObjectName createMBeanName(String domain, String name) throws MalformedObjectNameException {
+    final StringBuilder objName = new StringBuilder(domain);
+    objName.append(':');
+    objName.append("name=").append(name);
+    return new ObjectName(objName.toString());
+  }
```

Comment:
Couldn't we move these methods in `ServerJMX` and make them non static (eventually default) ? 
Do we really need to know about the impl here or its something that could be moved to the `ServerJMX` impl ?

Reply:
This is convenience and backwards compatibility.  Implementors used the terracotta MBean scheme from a long ways back for naming and implementation.  This should not be a requirement any longer.  This object naming is here to map those old calls.